Libbna/CUSTOM-CMS
# #239 solved the issue with image height on mobile view

To test: 
 Go to home page and try to increase and decrease the height and width of the page. 
 The images should not change its height.
